### PR TITLE
Adding new Categories Tab 

### PR DESCRIPTION
**Pull request by remya-james**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/FPET-251

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/test/fixtures/jsonDefinitions/Categories [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[
```
Comment (linusnorton) [inline src/test/fixtures/jsonDefinitions/Categories:1]: This file probably needs a .json extension

File: src/test/json2xlsx.js [hunk 2]
```diff
@@ -173,10 +174,6 @@ describe('json2xlsx', () => {
           assertCell(sheetName, 'N5', 'N');
           assertCell(sheetName, 'O5', 'Y');
         }
-        if (sheetName !== 'SearchAlias') { // SearchAlias tab uniquely does not have live from / to columns
```
[the remark this answers is not in the document]
Comment (remya-james) [reply src/test/json2xlsx.js:176]: Won't it fail at other sheets then?

[the remark this answers is not in the document]
Comment (remya-james) [reply src/test/json2xlsx.js:176]: I have tried that and it is failing at another sheet:
image

[the remark this answers is not in the document]
Comment (linusnorton) [reply src/test/json2xlsx.js:176]: looks like you've changed the assertions - what happens if you revert lines 178 and 179?

**Pull request merged**